src/Symfony/Component/Serializer/Normalizer/DenormalizerInterface.php

```diff
@@ -27,6 +27,13 @@
      * @param array  $context options available to the denormalizer
      *
      * @return object
+     *
+     * @throws \Symfony\Component\Serializer\Exception\BadMethodCallException
+     * @throws \Symfony\Component\Serializer\Exception\InvalidArgumentException
+     * @throws \Symfony\Component\Serializer\Exception\UnexpectedValueException
+     * @throws \Symfony\Component\Serializer\Exception\ExtraAttributesException
+     * @throws \Symfony\Component\Serializer\Exception\LogicException
+     * @throws \Symfony\Component\Serializer\Exception\RuntimeException
```

Comment:
throwing an exception is not a mandatory when you implement an interface. For a simple reason, exception is not a mandatory output, it can happen, or not.